src/index.html

```diff
@@ -8,5 +8,38 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
you must have possibility to manipulate your calendar by changing these class modificators  
```suggestion
    <div class="calendar calendar--start-day--wed calendar--month-length--31">
```